keras_nlp/models/xlm_roberta/xlm_roberta_tokenizer.py

```diff
@@ -123,10 +123,20 @@ def id_to_token(self, id):
 
     def token_to_id(self, token):
         """Convert a string token to an integer id."""
+
         if token in self._vocabulary_prefix:
             return self._vocabulary_prefix.index(token)
 
-        return int(self._sentence_piece.string_to_id(token).numpy()) + 1
+        spm_token_id = int(self._sentence_piece.string_to_id(token).numpy())
```

Comment:
let's compare in tensor space, convert to int at the end?

Reply:
Done!